schemes/musica/tuvx/musica_ccpp_tuvx.F90

```diff
@@ -128,7 +129,7 @@ subroutine tuvx_run( temperature, dry_air_density, &
       call calculate_heights( geopotential_height_wrt_surface_at_midpoint(i_col,:),  &
                               geopotential_height_wrt_surface_at_interface(i_col,:), &
                               surface_geopotential(i_col),                           &
-                              reciprocal_of_gravitational_acceleration,              &
+                              standard_gravitational_acceleration,                   &
```

Comment:
I would probably recommend calculating `reciprocal_of_gravitational_acceleration` outside of the `i_col` loop and then just passing it to `calculate_heights` like you were doing previously (this could potentially reduce the total number of divisions that have to be done).

Reply:
I agree, thank you for the comment!